File: Plugins/Aspose-Pdf-Java-for-Jython/asposepdf/WorkingWithDocumentObject/GetXMPMetadata.py
from asposepdf import Settings
from com.aspose.pdf import Document

class GetXMPMetadata:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithDocumentObject/GetXMPMetadata/'
        
        # Open a pdf document.
        doc = Document(dataDir + "input1.pdf")

        # Get properties
        print "xmp:CreateDate: "
        print doc.getMetadata().get_Item("xmp:CreateDate")
        print "xmp:Nickname: "
        print doc.getMetadata().get_Item("xmp:Nickname")
        print "xmp:CustomProperty: "
        print doc.getMetadata().get_Item("xmp:CustomProperty")
 
if __name__ == '__main__':        
    GetXMPMetadata()

File: plugins/Aspose_Pdf_Java_for_PHP/src/Aspose/Pdf/WorkingWithDocumentObject/GetXMPMetadata.php
<?php

namespace Aspose\Pdf\WorkingWithDocumentObject;

use com\aspose\pdf\Document as Document;

class GetXMPMetadata {

    public static function run($dataDir=null)
    {

        # Open a pdf document.
        $doc = new Document($dataDir . "input1.pdf");

        # Get properties
        print "xmp:CreateDate: " + $doc->getMetadata()->get_Item("xmp:CreateDate") . PHP_EOL;
        print "xmp:Nickname: " + $doc->getMetadata()->get_Item("xmp:Nickname") . PHP_EOL;
        print "xmp:CustomProperty: " + $doc->getMetadata()->get_Item("xmp:CustomProperty") . PHP_EOL;
    }

}